playbook/app/pb_kits/playbook/pb_icon/_icon.jsx

```diff
@@ -59,22 +78,30 @@ const Icon = (props: IconProps) => {
     size,
     spin = false,
   } = props
+
+  const faClasses = {
+    'fa-border': border,
+    'fa-fw': fixedWidth,
+    'fa-inverse': inverse,
+    'fa-li': listItem,
+    'fa-pulse': pulse,
+    'fa-spin': spin,
+    [`fa-${size}`]: size,
+    [`fa-pull-${pull}`]: pull,
+    [`fa-rotate-${rotation}`]: rotation,
+  }
+
+  // Lets check and see if the icon prop is referring to a custom Power icon...
+  const isCustomIcon = CUSTOM_ICONS.hasOwnProperty(icon)
+  // If so, then set fa-icon to "custom"
+  // this ensures the JS will not do any further operations
+  faClasses[`fa-${icon}`] = isCustomIcon ? 'custom' : icon
+
```

Comment:
Ultimately set the `fa-icon` attribute to `custom` if we detect the name exists in the imported mapping. Need to be careful with naming here or we will override the FA stuff. `e.g. "moon"`